### PR TITLE
Re-export types needed for the integration testing library.

### PR DESCRIPTION
**Pull request by DOBEN**
## Purpose

Re-export types needed for the integration testing library.

## Changes

Re-export types needed for account public key query and account signature verification.

### CODE REVIEW

File: concordium-smart-contract-testing/src/constants.rs [hunk 1]
```diff
@@ -28,9 +28,9 @@ pub(crate) fn contract_instance_query_account_keys_return_cost(num_keys: u32) ->
 
 /// Cost **in energy** of verification of an ed25519 signature.
 /// This should match the cost of
-/// [`verify_ed22519_cost`](concordium_smart_contract_engine::constants::verify_ed25519_cost)
-/// except the latter is the cost in interpreter energy, and this on is in
-/// [`Energy`].
+/// [`verify_ed22519_cost`](concordium_smart_contract_engine::constants::
+/// verify_ed25519_cost) except the latter is the cost in interpreter energy,
```
Comment (limemloh) [inline concordium-smart-contract-testing/src/constants.rs:32]: Seems weird if the formatter changed this. Did you add a newline here by mistake?

File: concordium-smart-contract-testing/src/types.rs [hunk 2]
```diff
@@ -36,7 +36,7 @@ pub(crate) struct ChainParameters {
     /// Euro per Energy ratio.
     // This is not public because we ensure a reasonable value during the construction of the
     // [`Chain`].
-    pub(crate) euro_per_energy: ExchangeRate,
+    pub(crate) euro_per_energy:    ExchangeRate,
```
Comment (limemloh) [inline concordium-smart-contract-testing/src/types.rs:39]: Also here, did we change the formatting?

**Review by limemloh — APPROVED** [review]

File: concordium-smart-contract-testing/src/constants.rs (hunk 1 above)
Comment (abizjak) [reply concordium-smart-contract-testing/src/constants.rs:32]: I think this will break the link, so this should be reverted or fixed some other way.

**Review by abizjak — APPROVED** [review]
If you can fix that link that was split into two lines then this looks good.

**Pull request merged**